File: GameOfDrones.Models/PlayerMove.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GameOfDrones.Models
{
    public class PlayerMove 
    {
        public int PlayerMoveId { get; set; }

        public int? MoveId { get; set; }
        public Move Move { get; set; }

        public int? PlayerId { get; set; }
        public Player Player { get; set; }
    }
}

File: GameOfDrones.UI/src/models/PlayerMove.ts
import { Player } from './Player';
import { Move } from './Move';

export class PlayerMove {
	gameId: number;
	roundId: number;
	moveId: number;
	playerId: string;

	player: Player;
	move: Move;
	constructor(player: Player, roundId?: number, playerId?: string, moveId?: number) {
		this.player = player;

		this.roundId = roundId;
		this.playerId = playerId;
		this.moveId = moveId;
	 }
}
